spec/features/venues/view_venue_details_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+feature 'View details of the venue' do
+  let!(:venue) { FactoryGirl.create(:venue) }
+
+  scenario 'When a user clicks on the venue name see venue details.' do
+    visit venues_path
+
+    click_link(venue.name)
+
+    expect(page).to have_content(venue.name)
```

Comment:
I would check for a couple details of the venue that would not also be on the venue index page. Throw in an expectation for one of the venue details besides name here